src/Client.js

```diff
@@ -158,8 +158,7 @@ var values = require('./values')
  *   Sets the maximum amount of time (in milliseconds) for query execution on the server
  * @param {?number} options.http2SessionIdleTime
  *   Sets the maximum amount of time (in milliseconds) an HTTP2 session may live
- *   when there's no activity. Must either be a non-negative integer, or Infinity to allow the
- *   HTTP2 session to live indefinitely (use `Client#close` to manually terminate the client).
+ *   when there's no activity. Must either be a non-negative integer.
```

Comment:
Clarify the documentation here.

```suggestion
 *   when there's no activity. Must be a non-negative integer, with a maximum value of 5000.
 *   If an invalid value is passed a default of 500 ms will be applied. If a value exceeding 5000 ms
 *  is passed (e.g. Infinity) the maximum of 5000 ms will be applied.
```